08_AdvancedTypes/UniversalTablePrinter.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Reflection;
+using System.Text;
+
+public class UniversalTablePrinter
+{
+    private readonly IEnumerable<object> _data;
+    private const int _columnLength = 20;
+
+    public UniversalTablePrinter(IEnumerable<object> data)
```

Comment:
You probably need that all objects in this IEnumerable be of the same type, you can use generics for that
```csharp
class UniversalTablePrinter<T>
{
   public UniversalTablePrinter(IEnumerable<T> data) 
//...
```